Task2/find_phones.py

```diff
@@ -0,0 +1,72 @@
+"""This module find phones:
+find in the text the numbers that correspond to the providers
+that were specified as command line arguments
+and create an output CSV file phones.csv"""
+
+import csv
+import re
+import argparse
+
+parser = argparse.ArgumentParser(description="Find phones")
+
+parser.add_argument("-i", action="store", dest="i",

```

Comment:
Please use long destination names (instead of single-letter), like input, cell_codes, provider, output.

Reply:
I have changed the parameters name